packages/autocomplete-core/rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import resolve from '@rollup/plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+import replace from '@rollup/plugin-replace';
+import { name } from './package.json';
+
+const createPlugins = ({ format }) => [
+  replace({
+    __DEV__: format === 'umd' ? false : 'process.env.NODE_ENV !== "production"',
+  }),
+  resolve({
+    extensions: ['.js', '.ts'],
+  }),
+  babel({
+    exclude: 'node_modules/**',
+    extensions: ['.js', '.ts'],
+  }),
+];
+
+export default [
+  {
+    input: 'src/index.ts',
+    output: {
+      file: 'dist/es/index.js',
+      format: 'es',
+      sourcemap: true,
```

Comment:
I believe `sourcemap` only makes sense for UMD.

Reply:
d6b5004